### ISSUE TITLE
Color contrast for some links is too low in light mode

### ISSUE THREAD

**Issue by RunDevelopment** [TypeStrong/typedoc#2228]
## Search terms

color contrast, light theme

## Expected Behavior

Links should have readable colors with at least 4.5:1 (WCAG AA) contrast.

## Actual Behavior

Several links in light mode have low contrast. Examples:
![image](https://user-images.githubusercontent.com/20878432/230780563-39cd669e-edfa-4f4c-bf8a-e35113ff7f73.png)
![image](https://user-images.githubusercontent.com/20878432/230780576-e9c89a14-fa90-40af-b2d6-0bfc60257b7f.png)
![image](https://user-images.githubusercontent.com/20878432/230780586-dfbfbf19-b06b-402b-bc9f-1d5dc19b9154.png)
![image](https://user-images.githubusercontent.com/20878432/230780626-560f5a7d-76fb-4fa5-a895-f71fca79b646.png)

## Steps to reproduce the bug

Open any doc page with links.

## Environment

-   Typedoc version: 0.24.1
-   TypeScript version: any
-   Node.js version: any
-   OS: any

**Comment by Gerrit0** [comment]
Wow, actually making this contrast meet 4.5 contrast makes everything so incredibly muted :/
